src/backend/distributed/sql/udfs/create_distributed_function/latest.sql

```diff
@@ -1,11 +1,15 @@
+DROP FUNCTION create_distributed_function(regprocedure, text, text);
+
 CREATE OR REPLACE FUNCTION create_distributed_function(function_name regprocedure,
 						       distribution_arg_name text DEFAULT NULL,
-						       colocate_with text DEFAULT 'default')
+						       colocate_with text DEFAULT 'default',
+						       force_pushdown bool DEFAULT NULL)
```

Comment:
small check on the name: Internally we usually talk about "function/procedure call delegation", since it's quite descriptive. We call it "distributed function" in user-facing APIs, mainly in a (poor) attempt to make the connection with distributed tables and co-location. This introduces another term: (function) pushdown. I do feel like this is more "pushdown" than "delegation", so perhaps it makes sense. We probably should rename/alias create_distributed_function at some point.

do you have any thoughts on the best name for this flag? I don't necessarily have a better idea.

Reply:
For now, changed the force_pushdown to force_delagation to keep it homogenous.

Reply:
We need to think something in the lines of "create_colocation_function()" / create_distributed_colocation_function()